enterprise/frontend/src/embedding-sdk/components/public/CreateQuestion/CreateQuestion.tsx

```diff
@@ -7,7 +7,7 @@ export type CreateQuestionProps = Partial<
   Omit<InteractiveQuestionProps, "questionId" | "children">
 >;
 
-/** @deprecated Use `InteractiveQuestion` without specifying `questionId` and with `isSaveEnabled={true}` instead. */
+/** @deprecated Use `<InteractiveQuestion questionId="new" />` instead. */
```

Comment:
Simplifies the deprecation messages as `isSaveEnabled` is true by default.

Reply:
good catch